controllers/fenceagentsremediation_controller_test.go

```diff
@@ -140,14 +142,15 @@ func buildFarPod() *corev1.Pod {
 func cliCommandsEquality(far *v1alpha1.FenceAgentsRemediation) (bool, error) {
 	//fence_ipmilan --ip=192.168.111.1 --ipport=6233 --username=admin --password=password --action=status --lanplus
 	if mocksExecuter.command == nil {
-		return false, errors.New("executedCommand is null")
+		return false, errors.New("The command from mocksExecuter is null")
 	}
-	expectedCommand, err := buildFenceAgentParams(far)
+	_, err := buildFenceAgentParams(far)
```

Comment:
I don't think you need this, it's tested in the Funtionionality block already?

Reply:
Correct, it is redundant 👍🏻